devsetup/scripts/gen-edpm-compute-node.sh

```diff
@@ -186,12 +186,9 @@ if [ ! -f ${DISK_FILEPATH} ]; then
     fi
     qemu-img create -o backing_file=${CRC_POOL}/centos-9-stream-base.qcow2,backing_fmt=qcow2 -f qcow2 "${DISK_FILEPATH}" "${EDPM_COMPUTE_DISK_SIZE}G"
     if [[ ! -e /usr/bin/virt-customize ]]; then
-        if [[ $(awk '{print $6}' /etc/redhat-release) =~ ^8.* ]] || \
-           [[ $(awk '{print $4}' /etc/centos-release) =~ ^8.* ]]; then
+        if [ $(rpm -E %{rhel}) -eq 8 ]; then
             sudo dnf -y install hexedit libguestfs-tools-c
```

Comment:
i think would be better to remove rhel checks and and just do sudo dnf -y install /usr/bin/virt-customize ?
hexedit should be automatically installed with virt-customize as it's a dep or there is some issue not sure why it was originally added

Reply:
It's something i was thinking of yesterday... I'd choose this approach too. @slagle, @fultonj ?

Reply:
`virt-customize` is provided by different packages on RHEL8 vs RHEL9. On my RHEL8 system I needed to add `hexedit`; the dep wasn't working to pull it in and I didn't dig into why.

https://github.com/openstack-k8s-operators/install_yamls/commit/fb31098ba71ec8244cf22e4e94c39b6fb2b170a3

If you can `dnf install /path/to/file` and have it identify the correct package relative to it's OS and confirm this works on all relevant platforms (cent-8 cent-9 rhel-8 rhel-9) then I'm OK with it. I did a lot of patches to devsetup/scripts/ because people only tested on rhel9 along the way but if you test it and it doesn't break, then that's fine with me.

Reply:
I see this example in DNF's docs so it sounds like a good approach.

`dnf install /usr/bin/rpmsign`: Install a package that provides the /usr/bin/rpmsign file.

https://dnf.readthedocs.io/en/latest/command_ref.html#install-examples

Reply:
I'll propose a follow-up patch for that.